annotations/models.py

```diff
@@ -1143,3 +1142,14 @@ class DocumentPosition(models.Model):
     If :attr:`.position_type` is :attr:`.WHOLE_DOCUMENT`\, then this can be
     blank.
     """
+
+class ImportTasks(models.Model):
+    """
+    Records the event that a set of :class:`.RelationSet`\s are accessioned to
+    Quadriga.
+    """
+    created = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey('VogonUser')
+    file_name = models.CharField(max_length=255, blank=False, null=False)
```

Comment:
there are filenames that are longer than that... (not sure why people do that but it happens)